### PR TITLE
Fixing grid images causing ffmpeg to crash

### PR DESCRIPTION
**Pull request by jmsether**
Fixing issue #38

### CODE REVIEW

File: iz_helpers/run.py [hunk 1]
```diff
@@ -149,7 +149,8 @@ def create_zoom_single(
             width,
             height,
         )
-        current_image = processed.images[0]
+        if(len(processed.images) > 0):
```
Comment (GeorgLegato) [inline iz_helpers/run.py:152]: what we do if no image ? could be exception?!
same for all occurances of "if len(processed)"...

**Review by GeorgLegato — CHANGES_REQUESTED** [review]

File: iz_helpers/run.py (hunk 1 above)
Comment (jmsether) [reply iz_helpers/run.py:152]: I tested it out extensively and Have yet to get it to fail. 
image

This is what it does right now if it's at zero and the web UI works as well.
image

Comment (jmsether) [reply iz_helpers/run.py:152]: @GeorgLegato this is mainly to fix what happened when you clicked interrupt on the web UI. I asked @v8hid if I could fix the problem with my patch and he gave the go-ahead. I can post the discord snippet if needed.

Comment (jmsether) [reply iz_helpers/run.py:152]: > what we do if no image ? could be exception?! same for all occurances of "if len(processed)"...

Also, why would we need this as an exception? This only happens when the user clicks interrupt. Right now I just having it return the black frame that was created at the start of the function.

**Pull request merged**